### PR TITLE
Feature/id 303/implementation of create operation

### PR DESCRIPTION
**Pull request by SamuelEscalera**
## Author
Samuel Escalera Herrera @SamuelEscalera 
## What?
Implement the create user operation in the database.
## Why?
To save the user's information and edit it
## CODE REVIEWER:
@DiegoFigueroaSevillano 
@VillalobosGarciaGaby 
## TRAINER
@mmoralesv

### CODE REVIEW

File: app/src/main/java/com/isc/hermes/model/User.java [hunk 1]
```diff
@@ -44,6 +44,13 @@ protected User(Parcel in) {
         id = in.readString();
     }
 
+    public User(String email, String fullName, String userName, String typeUser) {
```
Comment (DiegoFigueroaSevillano) [inline app/src/main/java/com/isc/hermes/model/User.java:47]: Javadoc

File: app/src/main/java/com/isc/hermes/database/AccountInfoManager.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package com.isc.hermes.database;
+
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+
+import com.isc.hermes.model.User;
+
+public class AccountInfoManager {
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/database/AccountInfoManager.java:9]: this class all method need javadoc and

File: app/src/main/java/com/isc/hermes/database/ApiRequestHandler.java [hunk 3]
```diff
@@ -44,6 +45,25 @@ public String getDataFromApi(String apiUrl) {
         return response.toString();
     }
 
+    public void postDataFromApi(String apiUrl, String jsonData) {
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/database/ApiRequestHandler.java:48]: this method need javadoc

**Review by VillalobosGarciaGaby — APPROVED** [review]

File: app/src/main/java/com/isc/hermes/database/AccountInfoManager.java (hunk 2 above)
Comment (SamuelEscalera) [reply app/src/main/java/com/isc/hermes/database/AccountInfoManager.java:12]: Resolved

File: app/src/main/java/com/isc/hermes/database/ApiRequestHandler.java (hunk 3 above)
Comment (SamuelEscalera) [reply app/src/main/java/com/isc/hermes/database/ApiRequestHandler.java:54]: Resolved

**Review by DiegoFigueroaSevillano — APPROVED** [review]
GOOD JOB!

**Pull request merged**